package-deploy.yaml

```diff
@@ -0,0 +1,129 @@
+tasks:
+  - task: docker.Version
+    configFilePath: scripts/configure_dev-deps.sh
+
+  - task: shell.docker.Ensure
+    name: deb
+    image: algorand/mule-linux-ubuntu
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/docker.ubuntu.Dockerfile
+    dependencies: docker.Version
+
+  - task: shell.docker.Ensure
+    name: rpm
+    image: algorand/mule-linux-centos
+    version: '{{ docker.Version.outputs.version }}'
+    dockerFilePath: docker/build/mule.go.centos.Dockerfile
+    dependencies: docker.Version
+
+  - task: docker.Make
+    name: generate-releases-page
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+    target: mule-deploy-generate_releases_page WORKDIR=/projects/go-algorand
+
+  - task: docker.Make
+    name: deb
+    docker:
+      image: algorand/mule-linux-ubuntu
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+      volumes: [
+        $HOME/packages:/root/packages,
+        $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent,
+        $HOME/.gnupg/pubring.kbx:/root/.gnupg/pubring.kbx
+      ]
+    target: mule-deploy-deb WORKDIR=/projects/go-algorand
+
+  - task: docker.Make
+    name: rpm
+    docker:
+      image: algorand/mule-linux-centos
+      version: '{{ docker.Version.outputs.version }}'
+      workDir: /projects/go-algorand
+      env: [
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      ]
+      volumes: [
+        $HOME/packages:/root/packages,
+        $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent,
+        $HOME/.gnupg/pubring.kbx:/root/.gnupg/pubring.kbx
+      ]
+    target: mule-deploy-rpm WORKDIR=/projects/go-algorand
+
+  - task: s3.BucketCopy
```

Comment:
are the src and dest fields backwards here? It looks like we're intending to download files from s3 and put them on our host. Also we should be using relative paths on the host since we'll be running mule from the root directory of our project. This way it will work on any system we need to run it on

Reply:
No, this package should already be on staging from running the previous build and package tasks.  We need to get it from staging and then put it in a location where we can add it to the previous packages and create a snapshot to then upload to `algorand-releases`.